chainerx/_docs/routines.py

```diff
@@ -1833,6 +1833,42 @@ def _docs_math():
 .. seealso:: :data:`numpy.bitwise_xor`
 """)
 
+    _docs.set_doc(
+        chainerx.left_shift,
+        """left_shift(x1, x2)
+Shift the bits of an integer to the left.
+
+Args:
+    x1 (~chainerx.ndarray or scalar): Input array of integers.
+    x2 (~chainerx.ndarray or scalar): Input array of integers.
+
+Returns:
+    :class:`~chainerx.ndarray`: Returned array: :math:`y = x_1 << x_2`
```

Comment:
In the context of the arithmetic formulae, `x1 << x2` means that `x1` is *much* less than `x2`.

Reply:
So should we change this to `y = x_1 * (2 ^ x_2)` ?

Reply:
@asi1024 Do you have any alternative? I think no one would be confused.

Reply:
It seems better to change to `y = x_1 2^{x_2})`, or write in natural language.

Reply:
I think ` x_1 << x_2` is a lot easier to understand.

Reply:
Even if so, `x1 << x2` should be written as a code snippet, not as an arithmetic formula.

Reply:
Hmm, I see. I agree with writing in natural language.